app/vulncode_app/views.py

```diff
@@ -24,10 +24,42 @@
 import json
 
 from django.http import HttpResponse
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
 
+from vulncode_app.serializers import ImpactedPackageSerializer
+from vulncode_app.models import ImpactedPackage
+from vulncode_app.models import Package
 from vulncode_app import api_data
 
 
+class VulnerabilityData(APIView):
+    def get(self, request, package_name):
+        vulnerability = []
+        extract = []
+        data = []
+
+        pk = Package.objects.filter(name=package_name)
+
+        for i, v in enumerate(pk):
```

Comment:
you are still enumerating pk. Instead loop on packages:
`for package in Package.objects.filter(name=package_name): ....`